src/DynamoCoreWpf/Views/Core/DynamoView.xaml

```diff
@@ -442,6 +442,13 @@
                                   Command="{Binding Path=NodeFromSelectionCommand}"
                                   Name="nodeFromSelection"
                                   InputGestureText="Ctrl + D" />
+                        <MenuItem Focusable="False"                                
+                                  Header="{x:Static p:Resources.NodesRunStatus}"
+                                  IsChecked="{Binding NodeRunChecked, Mode=OneWay}"
+                                  IsEnabled="{Binding NodeRunEnabled, Mode=OneWay}"
+                                  Command="{Binding Path=ComputeRunStateOfTheNodeCommand}"
```

Comment:
Having a checkable menu item corresponding to node state in the application's menus doesn't make any sense.